site/docs/v1/tech/developer-guide/api-gateways/aws-api-gateway.adoc

```diff
@@ -0,0 +1,217 @@
+
+image::api-gateways/aws-api-gateway/api-key.png[Enabling the login API,width=1200,role=bottom-cropped]
+
+Now create a test user to login with on FusionAuth. Navigate to [breadcrumb]#Users# in the sidebar and click the green `+` button at the top right of the screen to create a new user. Choose the appropriate tenant (usually `default`), and fill in the information. You can choose not to send an email to set up the password but rather set it directly when creating the user by toggling the [field]#Send email to set up password# switch. Record this user email and password, as we'll use it to obtain a token.
+
+After saving the user, click the [breadcrumb]#Registrations# tab. Click `Add registrations` and select the FusionAuth application you set up earlier.
+
+image::api-gateways/aws-api-gateway/registrations-tab.png[The application registrations tab,width=1200,role=bottom-cropped]
+
+Using cURL or Postman, make a POST request to your FusionAuth instance's `/api/login` endpoint. This will return a JWT in the response.
+
+The cURL call should look like this:
+
+[source,shell,title="cURL command to obtain a JWT token"]
+----
+curl --location --request POST 'https://<YOUR_FUSIONAUTH_URL>/api/login' \
+--header 'Authorization: <API_KEY>' \
+--header 'Content-Type: application/json' \
+--data-raw '  {
+    "loginId": "<USER_EMAIL>",
+    "password": "<USER_PASSWORD>",
+    "applicationId": "<APPLICATION_ID>",
+    "noJWT" : false
+  }'
+----
+
+Where:
+
+* `<YOUR_FUSIONAUTH_URL>` is the URL of your FusionAuth installation.
+* `<API_KEY>` is the API created above, enabled for login.
+* `<USER_EMAIL>` is the email address of the user added above.
+* `<USER_PASSWORD>` is the password of the user added above.
+
+The return response from FusionAuth should look similar to the following:
+
+[source,json,title="Return token from the API call"]
+----
+{
+    "token": "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImhDUjA4X3daR2s0OUFlYUFmRDY5ZmJKWmRGTSJ9.eyJhdWQiOiI2M2I3M2Y3Ni03NDAwLTQ4N2QtYjEyMi01NzA1Yjg0OGRhODAiLCJleHAiOjE2NzMzNjYyMDQsImlhdCI6MTY3MzM2MjYwNCwiaXNzIjoiaHR0cHM6Ly9mdXNpb25hdXRoLnJpdHphLmNvIiwic3ViIjoiMzk2MzAwMGYtNjg2ZC00MTY5LWI2MjgtOWM5YzQ1MzRiNzgwIiwianRpIjoiZDk3ZGIyZWYtZjExNS00ZDIxLWFlOTQtMDIyN2RmMGU4YzI5IiwiYXV0aGVudGljYXRpb25UeXBlIjoiUEFTU1dPUkQiLCJlbWFpbCI6ImJvYkBhd3MuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsInByZWZlcnJlZF91c2VybmFtZSI6ImJvYmF3cyIsImFwcGxpY2F0aW9uSWQiOiI2M2I3M2Y3Ni03NDAwLTQ4N2QtYjEyMi01NzA1Yjg0OGRhODAiLCJyb2xlcyI6W10sImF1dGhfdGltZSI6MTY3MzM2MjYwNCwidGlkIjoiZjAwNGMxZmUtNDg0Yi05MDJjLWQ3Y2EtYmRiYzQ2NGRhMGI3In0.m7gzXhNLToPNVE1p5Vo2pLgP6WBcPNfS_zZJnJ81mdEgi6-orViz-tU8j0L8wva0-8KlMdy54cq_XjnDnYJ0aX90O4ZE_QVU5NuDDfzXH14wQtKQoIIydsB6ZvQoBt8JNFUHJb9ANLCGnfn6FVQKqPIzye18Gx_7wYSVokw3eLNFyzrq9dwOD5Q8V9gvZmXV2pTokQAtA7qFaadb2dIeFlSEB7wamKiZLXILjeWAeMbbvAAMQZWFh46UJjwr06QTd8PxQmRwDWWznJy1Vs8EAgZA4vkRSWnn3IbiaCtOaL1ANuEex6il7q32ahxj0Ncm9wn0DbDsQE9NB0CCNTSIhA",
+    "tokenExpirationInstant": 1673366204805,
+    "user": {
+        ....
```

Comment:
```suggestion
```

Reply:
these look red and angry when syntax highlighed because they aren't valid json. Don't have a real good option here, so omit them.